upup/models/cloudup/resources/addons/networking.cilium.io/k8s-1.12-v1.8.yaml.template

```diff
@@ -886,7 +886,7 @@ spec:
             topologyKey: "kubernetes.io/hostname"
       containers:
         - name: hubble-relay
-          image: "docker.io/cilium/hubble-relay:{{ .Networking.Cilium.Version }}"
+          image: "{{ .HubbleRelayRepository }}:{{ .Networking.Cilium.Version }}"
```

Comment:
Out of curiosity, any reason to use `.Networking.Cilium.Version` and not `.Version` here also?

Reply:
I did a find and replace and missed that, thanks, done